bin/zopen-build

```diff
@@ -1832,13 +1883,42 @@ install()
 
       #TODO: Hack so that we can use coreutils md5sum without impacting builds
       ZOPEN_DEPS="${ZOPEN_DEPS} coreutils jq"
+      if [ -n "${signPax}" ]; then
+              if [ -z "${ZOPEN_GPG_SECRET_KEY_FILE}" ] || [ -z "${GPG_SECRET_KEY_PASSPHRASE}" ] || [ -z "${GPG_PUBLIC_KEY_PATH}" ]; then
+                        printWarning "GPG keyring environment variables are not set!"
+              else
+                ZOPEN_DEPS="${ZOPEN_DEPS} gpg"
+              fi
+      fi
       setDepsEnv
 
       pax_size=$(getPathSizeInBytes "${paxFileName}")
       md5sum=$(md5sum "${paxFileName}" | awk '{print $1}')
-      jq '.product |= . + { "pax": "'${ZOPEN_NAME}.${LOG_PFX}.zos.pax.Z'", "pax_size": "'${pax_size}'", "md5": "'${md5sum}'" }' "${ZOPEN_ROOT}/install/metadata.json" > "${ZOPEN_ROOT}/install/metadata.json.tmp"
+
+      if [ -n "${signPax}" ]; then
+        if [ -z "${ZOPEN_GPG_SECRET_KEY_FILE}" ] || [ -z "${GPG_SECRET_KEY_PASSPHRASE}" ] || [ -z "${GPG_PUBLIC_KEY_PATH}" ]; then
+                printWarning "GPG keyring environment variables are not set!"
```

Comment:
Can you consolidate this check with the one starting at line 1886? Perhaps use a boolean to indicate all the reqs are satisfied

Reply:
Thinking about this more, if --sign-pax is specified, we should probably emit an error message. The user can just omit the --sign-pax option to bypass it.